README.md

```diff
@@ -7,6 +7,44 @@ This meta-package contains the following packages:
 * depthai_ros_msgs
 * node_interface
 
+# Setup
+## Requirements
+Though this metapackage might work on other systems, it has only been tested on:
+* Ubuntu 18.04 LTS
+* Ubuntu 20.04 LTS
+
+A list of pre-requisites:
+* C++ compiler (recommended: `g++` or `clang` with at least C++17 support)
+* CMake (3.10.2 or higher)
+* Boost (1.65.1 or higher)
+* OpenCV (3.2 or higher)
+* ROS (melodic or higher)
+* ROS Packages (see `rospack deps depthai_ros_driver`)
+
+## Grab the source code
+To use `depthai_ros` metapackage, clone this repo with submodules:
+```
+$ git clone --recursive https://github.com/rapyuta-robotics/depthai_ros.git
+```
+For older versions of git, one of the following processes might be needed:
+* Use `--recurse-submodules` instead of `--recursive`
+* Clone without any flags and then grab the submodules using `git submodule update --init --recursive`
+
+# Usage
+0. Compile using `cmake build depthai_ros` or your preferred `cmake_build` or `colcon` process
+1. Launch the node or nodelet by appropriately choosing the correct launch file and parameters:
```

Comment:
model files `.blob` and `.json` are necessary for running this default launch file.

Reply:
Yes, I'll add that information as well.